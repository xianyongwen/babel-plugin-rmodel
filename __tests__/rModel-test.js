
const babel = require('babel-core');
const plugin = require('../src/index.js');

var example = `
React.createElement("input", {
  rModel: this.state.counter
})
`;

it('rModel', () => {
  const {code} = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
});