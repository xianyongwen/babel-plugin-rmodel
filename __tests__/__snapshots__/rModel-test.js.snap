// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rModel 1`] = `
"
React.createElement(\\"input\\", {
  value: this.state.counter,
  onChange: e => {
    let temp = e.target.value;
    this.setState(state => ({
      counter: temp
    }));
  }
});"
`;
